models/comments.model.js

```diff
@@ -26,4 +27,18 @@ async function createComment(articleId, author, body) {
 }
 
 
-module.exports = {selectCommentsByArticleId, createComment}
\ No newline at end of file
+async function deleteCommentById(id) {
+    if (!Number.isInteger(id)) {
+        return Promise.reject({ status: 400, msg: "Bad Request" })
```

Comment:
Custom errors are a great opportunity to be really specific with the error messages. You will be the user of this API in frontend review week so create helpful error messages.